website/static/js/addProjectPlugin.js

```diff
@@ -38,6 +38,7 @@ var AddProject = {
         self.showMore = m.prop(false);
         self.newProjectName = m.prop('');
         self.newProjectDesc = m.prop('');
+        self.newProjectStorageLocation = m.prop(window.contextVars.storageRegions[0]); // first storage region is default
```

Comment:
When creating a project from the "My Projects" page, this default is correct (the user default). However, when creating a component via the "Add Component" button on the project overview page, the default should be the parent node's region (not the user default).

Reply:
👍